frontend/test/metabase/scenarios/question/custom_column.cy.spec.js

```diff
@@ -692,4 +692,34 @@ describe("scenarios > question > custom columns", () => {
     cy.contains("37.65");
     cy.findByText("No discount");
   });
+
+  it.skip("should work with relative date filter applied to a custom column (metabase#16273)", () => {
+    cy.intercept("POST", "/api/dataset").as("dataset");
+
+    openOrdersTable({ mode: "notebook" });
+    cy.findByText("Custom column").click();
+    popover().within(() => {
+      cy.get("[contenteditable='true']")
```

Comment:
What would you say of creating a chainable function with a more declarative name for selecting the custom expression editor?

Reply:
Will do.
So far I'm still waiting for some larger patterns to emerge to avoid premature optimizations and abstractions. Keep in mind a few more things:
- we're going to divide and rearrange a lot of the files, so it's also tricky to know which test will go where (I can always create a global helper function, but I try to think carefully before doing that)
- depending on the "entry point", these steps can differ a lot
    - simple/custom question
    - `/question/:id/notebook`
    - `openFooTable({mode])`
    - there's probably more but in general, UI is different

One more thing - ideally, for repros we don't even touch UI for the set up phase. Unfortunately, until these issues get resolved I am forced to use UI instead of API, but it's quite possible I'll end up rewriting all these tests using just the API in the future.

The logic is - have a set of smoke tests that are going to test a certain UI flow ONCE, use API for the repros and everything else. There's really no added benefit in going through the same "motion" over and over again.

Hope it makes sense?

Reply:
Yes. 

I am more a believer in continuous small improvement than "soon we will make big improvements".

Not that I don't believe the latter, it works well sometimes.

Thanks!